website/src/footer-content.tsx

```diff
@@ -31,13 +31,16 @@ const footerStyle = makeStyles((theme: Theme) => ({
     },
     legalText: {
         fontWeight: theme.typography.fontWeightLight
+    },
+    toolbarManageCookies: {
+        fontWeight: theme.typography.fontWeightLight
     }
 }));
 
 const FooterContent: React.FunctionComponent<{ expanded: boolean }> = ({ expanded }) => {
     const classes = footerStyle();
     const theme = useTheme();
-    const isSmallDisplay = useMediaQuery(theme.breakpoints.down('sm'));
+    const isSmallDisplay = useMediaQuery(theme.breakpoints.down('lg'));
```

Comment:
we shouldn't change the definition of isSmallDisplay to encompass up to large displays, as this will trigger on many devices where it is not intentional. What was the reasoning behind this change?

Reply:
That variable is only to decide about the behaviour of the menu. Because I added the "Manage Cookies" item here, and the footer got too full for the lg size, I decided to switch to the collapsed menu in the footer at a larger state. Admittedly the naming of that variable is not suitable anymore.